### ISSUE TITLE
Sliders suddenly appear on left side after latest update

### ISSUE THREAD

**Issue by bubffm** [oznu/homebridge-config-ui-x#2261]
### Describe The Bug

Some UI inconsistancies (sliders) have been introduced with the latest update. See screenshot. I found no way to get rid of them.

Thanks for checking

<img width="554" alt="Bildschirmfoto 2024-11-30 um 17 54 18" src="https://github.com/user-attachments/assets/a7305ff8-0e7c-440e-a297-64bf2427654c">

### Homebridge UI Version

4.64.0

### Homebridge Version

1.8.5.

### Node.js Version

22.11.0

### Operating System

Official Homebridge Raspberry Pi Image

### Environment Info

None

### Raspberry Pi Model

None

**Comment by bwp91** [comment]
Which platform / browser are you viewing the UI from?

**Comment by bubffm** [comment]
Safari on iPad / Mac

Am Sa., 30. Nov. 2024 um 18:10 Uhr schrieb Ben ***@***.***>:

**Comment by bwp91** [comment]
which version of macOS and safari? asking because i am on the latest versions of these and I am not seeing these scroll bars

**Comment by bubffm** [comment]
Yes, I got the latest too.  18.1.1. on iOS and 15.1.1. on MacOS.  Strange...

**Comment by bubffm** [comment]
Just found the core of the issue: 
I have an "App" created from the Safari Tab containing the homebridge link both with iOS and MacOS.  Opening homebridge in "plain" Safari creates no sliders. 

I then deleted the "Safari App" and created a new one.  But again those sliders appear... 

Slightly puzzled what causes this....

**Comment by bubffm** [comment]
The issue has disooaeared with the ltest version. Whatever this was....

Thanks
